pt-BR/getting_started.md

```diff
@@ -541,32 +546,36 @@ file, and replace its contents with:
 </ul>
 ```
 
-The above code is a mixture of HTML and *ERB*. ERB is a templating system that
-evaluates Ruby code embedded in a document. Here, we can see two types of ERB
-tags: `<% %>` and `<%= %>`. The `<% %>` tag means "evaluate the enclosed Ruby
-code." The `<%= %>` tag means "evaluate the enclosed Ruby code, and output the
-value it returns." Anything you could write in a regular Ruby program can go
-inside these ERB tags, though it's usually best to keep the contents of ERB tags
-short, for readability.
-
-Since we don't want to output the value returned by `@articles.each`, we've
-enclosed that code in `<% %>`. But, since we *do* want to output the value
-returned by `article.title` (for each article), we've enclosed that code in
-`<%= %>`.
-
-We can see the final result by visiting <http://localhost:3000>. (Remember that
-`bin/rails server` must be running!) Here's what happens when we do that:
-
-1. The browser makes a request: `GET http://localhost:3000`.
-2. Our Rails application receives this request.
-3. The Rails router maps the root route to the `index` action of `ArticlesController`.
-4. The `index` action uses the `Article` model to fetch all articles in the database.
-5. Rails automatically renders the `app/views/articles/index.html.erb` view.
-6. The ERB code in the view is evaluated to output HTML.
-7. The server sends a response containing the HTML back to the browser.
-
-We've connected all the MVC pieces together, and we have our first controller
-action! Next, we'll move on to the second action.
+O código acima é uma mistura de HTML e *ERB*. ERB é um sistema de _template_ que
+avalia código Ruby embarcado em um documento. Aqui, podemos ver dois tipos de
+_tags_ ERB: `<% %>` e `<%= %>`. A _tag_ `<% %>` significa "avaliar o código Ruby
+incluso". A _tag_ `<%= %>` significa "avaliar o código Ruby incluso e retornar o
+valor de saída". Qualquer coisa que possa ser escrita em um programa normal em
+Ruby pode ir dentro dessas _tags_ ERB, embora geralmente seja melhor manter o
+conteúdo das _tags_ ERB de forma curta para facilitar a leitura.
+
+Já que não queremos gerar o valor retornado por `@articles.each`, vamos colocar
+esse código em `<% %>`. Porém, uma vez que *queremos* exibir o valor retornado
```

Comment:
Como a gente usa muito itálico será que podemos marcar esse queremos como negrito pra diferenciar? Ou acha demais?

Reply:
> Como a gente usa muito itálico será que podemos marcar esse queremos como negrito pra diferenciar? Ou acha demais?

Boa, achei uma ótima ideia! Feito em f74e615c21c74c49489119059c03b27fb171a88e